sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -0,0 +1,20 @@
+namespace Azure.Monitor.Ingestion
+{
+    public partial class LogsIngestionClient
+    {
+        protected LogsIngestionClient() { }
+        public LogsIngestionClient(System.Uri endpoint, Azure.Core.TokenCredential credential) { }
+        public LogsIngestionClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.Monitor.Ingestion.LogsIngestionClientOptions options) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+        public virtual Azure.Response Upload(string ruleId, string streamName, Azure.Core.RequestContent content, string contentEncoding = null, Azure.RequestContext context = null) { throw null; }
```

Comment:
Do we call it just `ruleId` in the other languages?  It's not a problem to go out with this for preview, but we should make sure it's aligned before GA.  Given all the concepts in the service, `dataCollectionRuleId` might be a more helpful name.